src/System.Threading.Tasks.Extensions/src/System.Threading.Tasks.Extensions.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(NetFrameworkMinimum);netstandard2.0;netcoreapp2.1</TargetFrameworks>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <StrongNameKeyId>Open</StrongNameKeyId>
+    <IsPackable>false</IsPackable>
+    <VersionPrefix>4.5.4</VersionPrefix>
+    <VersionPrefix Condition="'$(IsPackable)' == 'true'">4.6.0</VersionPrefix>
+    <AssemblyVersion>4.2.0.1</AssemblyVersion> <!-- NO-INCREMENT: This version is frozen for .NET Standard and .NETCoreApp because the assembly ships inbox. -->
+    <AssemblyVersion Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">4.2.0.1</AssemblyVersion>
+    <AssemblyVersion Condition="'$(IsPackable)' == 'true' and $([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">4.2.1.0</AssemblyVersion>
+    <PackageValidationBaselineVersion>4.5.4</PackageValidationBaselineVersion>
+    <IsPlaceholderTargetFramework Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETCoreApp'">true</IsPlaceholderTargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) != '.NETCore'" Include="System.Runtime.CompilerServices.Unsafe" VersionOverride="4.5.3" />
```

Comment:
Why the VersionOverride?

Reply:
The version specified in Directory.Packages.props is 6.0.0. If I let this project use that version, I get this error:

```
C:\Users\calope\.nuget\packages\system.runtime.compilerservices.unsafe\6.0.0\buildTransitive\netcoreapp2.0\System.Runtime.CompilerServices.Unsafe.target
s(4,5): error : System.Runtime.CompilerServices.Unsafe doesn't support netcoreapp2.1. Consider updating your TargetFramework to netcoreapp3.1 or later.
[D:\maintenance-packages\src\System.Threading.Tasks.Extensions\src\System.Threading.Tasks.Extensions.csproj::TargetFramework=netcoreapp2.1]
```

So I opted to using the version the package was originally using in the old repo.

Do you have an alternative suggestion to fix that error?

Reply:
I'm confused. System.Threading.Tasks.Extensions shouldn't reference System.Runtime.CompilerServices.Unsafe when targeting netcoreapp2.1. That's what the condition is for.

Reply:
What do you mean by not found? The version is on nuget.org and on our own feeds. Can you please elaborate and post the error?

Reply:
If I removed the VersionOverride, the error was the one I shared in my comment above: https://github.com/dotnet/maintenance-packages/pull/38#discussion_r1470403263

But I removed VersionOverride again, and now it's succeeding. I do not understand what changed.

I'll push a commit to remove the VersionOverride.